### DISCUSSION TITLE
Query regarding github insights and best practice for merging

### DISCUSSION THREAD

**Discussion by th3bossc** [manojnpalat/manojnpalat.github.io#7] [General]
Sir, I had a query regarding how the individual contributions of each team member will be evaluated.
As of now we have mostly worked in separate branches for every single issue / sub issue.
Due to this, the contributions section in github insights, which only show the contributions to the `main` branch, shows only my contribution, when I pushed to main directly while setting up the project. As the remaining team members, and myself since then, have worked on separate branches, the pull requests and our later commits don't show up in the github insights, and I am concerned whether this will cause an issue in the grading process.

I also wanted to ask if it is considered best practice for one team member to be in charge of merging the pull requests issued by the other members after reviewing, or is it better for each member to merge their work by themselves.

Thanks in advance
Diljith P D (SE LAB 2024)
[github repo for reference](https://github.com/th3bossc/NITCONF-SE_LAB)

**Comment by manojnpalat** [comment]
@th3bossc  The best practice is to merge the pull requests by the respective authors into the main repository. Since that was not the case now, what you can do is to assign the PRs to the respective people who have worked on the same. Will discuss with Professor Joe and Professor Anu on the same.
